src/CiviDistManagerBundle/Controller/CheckController.php

```diff
@@ -200,6 +203,7 @@ protected function getLatestStable() {
         'civicrm-drupal@6.x' => array('commit' => "6.x-$rev"),
         'civicrm-drupal@7.x' => array('commit' => "7.x-$rev"),
         'civicrm-drupal@8.x' => array('commit' => "8.x-$rev"),
+        'civicrm-standalone' => array('commit' => $rev),
```

Comment:
The `git` array defines the commits/tags that correspond to the code for a particular build. We're not actually tagging `civicrm-standalone.git` (and it's not used as part of that build). 

I'm not sure that tags should be sync'd, since `civicrm-standalone.git` is more like a project-template rather than a library that one depends on. If we don't do those tags, then we shouldn't have this line.

But if you think there's a reason why we should, then this should be here. (But there are probably some other bits that will be required for that.)